client/client.go

```diff
@@ -420,28 +421,18 @@ func (r *NotaryRepository) Publish(getPass passwordRetriever) error {
 	if err != nil {
 		return err
 	}
-
+	update := make(map[string]json.RawMessage)
```

Comment:
Not sure if we should make more or less use of RawMessage, but in this specific case it is bleeding a bit into client.go that didn't really have explicit uses of it yet.